oneanddone/functional_tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think I'd prefer a path of `tests/functional` as it works well when we have different suites of tests.